extension/duckdb/src/include/duckdb_storage.h

```diff
@@ -8,7 +8,11 @@ namespace duckdb_extension {
 
 class DuckDBStorageExtension final : public storage::StorageExtension {
 public:
-    static constexpr const char* dbType = "DUCKDB";
+    static constexpr const char* DB_TYPE = "DUCKDB";
+
+    static constexpr const char* SKIP_UNSUPPORTED_TABLE_KEY = "SKIP_UNSUPPORTED_TABLE";
```

Comment:
Up to u. Do we want to maintain all extension config into its own class?

Reply:
Yes, these constants are only be used by duckdb extensions.